docs/components/forms.rst

```diff
@@ -105,7 +105,7 @@ To add a new field to your Form:
 
    - **Hidden** - This field won't be visible on the Form, but include default values, saved along with the Form submission, for reporting or internal tagging purposes.
 
-   - **List - Country** - This populates Mautic's default, non-editable country list. To use a custom list you should make use of the Select field type and manually enter the countries you would like to include.
+   - **Select: Country** - This populates Mautic's default, non-editable country list. To use a custom list you should make use of the Select field type and manually enter the countries you would like to include.
```

Comment:
```suggestion
.. vale off

   - **Select: Country** - This populates Mautic's default, non-editable country list. To use a custom list you should make use of the Select field type and manually enter the countries you would like to include.
   
  .. vale on
```

Vale is moaning about this so we can turn it off temporarily with this syntax. Just ensure that it has a line break above and below each statement (couldn't really see if that was the case with the suggestion). we can do that with each of the bullets below.

Reply:
Hm, I didn't know we had a test for it
Should I try to fit the guidelines or just disable?

Reply:
We can't, because the form is actually using the syntax stated in the dropdown option. Hence suggesting just turning Vale off for this section and then back on again.